web/templates/stream_settings/first_stream_created_modal.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="micromodal" id="first_stream_created_modal" aria-hidden="true">
+    <div class="modal__overlay" tabindex="-1">
+        <div class="modal__container" role="dialog" aria-modal="true"
+          aria-labelledby="first_stream_created_modal_label">
+            <header class="modal__header">
+                <h1 class="modal__title" id="first_stream_created_modal_label">
+                    <span>
+                        {{#tr}}
+                        Stream <z-stream></z-stream> created!
+                        {{#*inline "z-stream"}}<b>{{> ../inline_decorated_stream_name stream=stream}}</b>{{/inline}}
+                        {{/tr}}
+                    </span>
+                </h1>
+                <button class="modal__close" aria-label="{{t 'Close modal' }}" data-micromodal-close></button>
+            </header>
+            <main class="modal__content">
+                <span>
+                    {{t 'You will now see the stream you created. To go back to stream settings, you can:' }}
+                    <ul>
+                        <li>
+                            {{#tr}}
+                            Click on <z-stream></z-stream> at the top of your Zulip window.
+                            {{#*inline "z-stream"}}<b>{{> ../inline_decorated_stream_name stream=stream}}</b>{{/inline}}
+                            {{/tr}}
+                        </li>
+                        <li>
+                            {{#tr}}
+                            Use the <b>back</b> button in your browser or desktop app.
+                            {{/tr}}
+                        </li>
+                    </ul>
+                </span>
+            </main>
+            <footer class="modal__footer">
+                <button type="button" class="modal__btn dialog_exit_button animated-purple-button"
+                  aria-label="{{t 'Close this dialog window' }}" data-micromodal-close>{{t "Continue" }}</button>
+            </footer>
+        </div>
+    </div>
```

Comment:
You should see how we implement other modals using `dialog_widget`. You would only have to create the template containing the content of the modal.

Reply:
I've shifted to using dialog_widget. The template `first_stream_created_modal.hbs` now only contains the body of the modal.